scripts/templates/webdriverio.tpl.d.ts

```diff
@@ -248,6 +248,14 @@ declare namespace WebdriverIO {
             func: (origCommand: Function, ...args: any[]) => any,
             attachToElement?: boolean
         ): void;
+
+        /**
+         * create custom selector
+         */
+        addLocatorStrategy(
+            name: string,
+            func: (elementFetchingMethod: (selector: String) => any) => void
```

Comment:
`string` instead of `String`.

Is `selector` always string?

Reply:
> * Is `selector` always string?

Yes!